### PR TITLE
Merge upstream 2024.11.2

### PR DESCRIPTION
**Pull request by zatteo**
# Main changes by Bitwarden in the merge upstream

- Bug fixes
- CryptoService -> KeyService

# My changes after the merge upstream

- [fix: Remove unused translations](https://github.com/cozy/cozy-keys-browser/pull/339/commits/66f9bf6dd6aa87617a07e87751bce508f0949344)
- [refactor: Export icons differently to ease merge upstream](https://github.com/cozy/cozy-keys-browser/pull/339/commits/8262e4ffe6a91bfc21d469cf02a64bf2d5291efe)
- [feat: Update cozy packages](https://github.com/cozy/cozy-keys-browser/pull/339/commits/8fdb2eddc5d31ef684ea621d6f22a66feb3c2d0c)
- [fix: Hide biometrics and 2FA in settings](https://github.com/cozy/cozy-keys-browser/pull/339/commits/2d9b37b6691cadded40f6e883f67a85a09a7c863)
- [feat: Rewrite package-lock.json](https://github.com/cozy/cozy-keys-browser/pull/339/commits/39251ae3a3178fe788bdebe1ffd6b703097eb981)
- [fix: Lint and missing translations](https://github.com/cozy/cozy-keys-browser/pull/339/commits/6720b82c86c6916640109c59b1a5191b82b5e3a0)
- [fix: Remove bitwarden wording](https://github.com/cozy/cozy-keys-browser/pull/339/commits/b0f352e7d1f2ca86a4e5423c894af99798a43cfc)
- [test: Fix tests after merge upstream](https://github.com/cozy/cozy-keys-browser/pull/339/commits/8ac92c2baefed8fb7d4892f5859ea8fd28768c28)

### CODE REVIEW

File: apps/browser/src/autofill/utils/svg-icons.ts
Comment (Merkur39) [inline apps/browser/src/autofill/utils/svg-icons.ts:1]: Is this change really useful?

File: package-lock.json
Comment (Merkur39) [inline package-lock.json:1]: Is it normal to have to do this?
I don't remember it from previous merge-upstreams.

**Review by Merkur39 — APPROVED** [review]
LGTM 👍

File: package-lock.json
Comment (zatteo) [reply package-lock.json:1]: I can't say it is normal to do this, I got a very strange error "Error: Cannot find module 'npmlog'" fixed by rewriting package-lock.json.

File: apps/browser/src/autofill/utils/svg-icons.ts
Comment (zatteo) [reply apps/browser/src/autofill/utils/svg-icons.ts:1]: Bitwarden started with an object export `export { ... }` and then switched to multiple `export const ...`. We started like bitwarden by adding our icons in the object export, so here with switch like bitwarden. It is a little better when doing a merge upstream to have everything using the same export.

**Pull request merged**